test/java/seedu/addressbook/commands/ViewCommandTest.java

```diff
@@ -49,7 +49,7 @@ public void execute_personNotInAddressBook_returnsPersonNotInAddressBookMessage(
         ReadOnlyPerson stranger = new Person(new Name("me"),
                                              new Phone("123", true),
                                              new Email("some@hey.go", true),
-                                             new Address("nus", false),
+                                             new Address("123, Clementi Ave 3, #12-34, 231534", false),
```

Comment:
Yes this is the correct formatting for separating parameters into multiple lines! :)